ckanext/switzerland/templates/scheming/form_snippets/organization.html

```diff
@@ -29,6 +30,10 @@
           {% endfor %}
         </select>
       </div>
+      {{- form.info(
+          text= _('Sichtbarkeit auf Veröffentlicht setzen, um diesen Datensatz automatisch zu veröffentlichen'),
```

Comment:
The word `automatisch` sounds a little confusing to me in this context.
Maybe something like `Sichtbarkeit auf Veröffentlicht setzen, um diesen Datensatz zu publizieren.` makes sense, too?

But we usually use english as default language in code :)